### DISCUSSION TITLE
OAuth: Add support for manual org assignment and oauth

### DISCUSSION THREAD

**Discussion by biomack** [grafana/grafana#66830] [General]
Can you add somethings like this https://github.com/grafana/grafana/issues/22605 ? Because we still have problem with mapping orgs in keycloak and you said here https://github.com/grafana/grafana/issues/24588 that we will get way how to merge all orgs, but we don't have anything.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This looks like it was completed, so I will close this discussion.  Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you need something else and would like to request the feature.
